brigade-worker/src/k8s.ts

```diff
@@ -46,7 +46,7 @@ const wrapClient = fns => {
   // wrap client methods with retry logic
   for (let fn of fns) {
     let originalFn = defaultClient[fn.name];
-    defaultClient[fn.name] = function() {
+    defaultClient[fn.name] = function () {
```

Comment:
I don't think there's an actual change here, right?
As it doesn't add anything, and we should probably add some linters for TypeScript and fix them everywhere (see #739), would it make sense to revert this type of changes in this file?

Reply:
this was most certainly caused by VS Code auto-formatter on cmd-s while I was editing on Mac. I'm now editing on Windows and, weirdly, ctrl-s does not format it, so I'll revert it (and I'm pretty sure that I haven't done any explicit JS configuration on VS Code settings, so who knows...)